server/routes/upload.js

```diff
@@ -0,0 +1,72 @@
+const Profile = require("../models/Profile");
```

Comment:
Some of the code in this PR is from the previous branch - please rebase/adjust the base of this PR 👍🏽

Reply:
I tried rebasing this branch and I am not sure if I did it correctly... This branch is a feature branch off of a feature branch and i tried git rebase <previous_feature_branch>

Reply:
@rajivtitus